actions/clone-template.go

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package actions
+
+import (
+	"log"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/eclipse/codewind-installer/utils"
+	"github.com/urfave/cli"
+)
+
+// CloneTemplate from github
+func CloneTemplate(c *cli.Context) {
+	var tempPath = ""
+
+	const GOOS string = runtime.GOOS
+	if GOOS == "windows" {
```

Comment:
Could the filepath package you've used below be used for this purpose? Just wondering, as path.join in node I think takes OS into account

Reply:
Not sure filepath can be used. The runtime is the safest way to determine the operating system in this case.